### PR TITLE
Update Tensorflow documentation tips section

### PR DESCRIPTION
**Pull request by tieujason330**

### CODE REVIEW

**Comment by tieujason330** [comment]
@LuQQiu @ZacBlanco PTAL - I made some minor changes in the tip section. Let me know if anything is incorrect. I mainly tried clearing up the phrasing for some sentences

File: docs/en/compute/Tensorflow.md [hunk 1]
```diff
@@ -156,35 +156,35 @@ script [tf_cnn_benchmarsk.py](https://github.com/tensorflow/benchmarks/blob/mast
...
-When running Tensorflow on top of HDFS, S3, and other under storages, it requires to 
-configure Tensorflow and modify tensorflow applications. Through Alluxio POSIX API,
-users only need to mount their various under storages to Alluxio once and mount the 
-parent folder of those under storages that containing training data to the local filesystem.
-After mounting, data in various under storages become immediately available through Alluxio
-fuse mount point and can be transparently accessed to Tensorflow applications.
-If the Tensorflow application has the data location parameter, we only need to pass the 
+Running Tensorflow on top of HDFS, S3, and other under storages could require different 
+configurations, making it difficult to manage and integrate Tensorflow applications with 
+different under storages. Through Alluxio POSIX API,
```
Comment (ZacBlanco) [inline docs/en/compute/Tensorflow.md]: Can we follow the line break guidelines in https://docs.alluxio.io/os/user/stable/en/contributor/Documentation-Conventions.html#line-breaks

mainly break line on new sentence. This makes diffing docs easier

File: docs/en/compute/Tensorflow.md [hunk 2]
```diff
@@ -156,35 +156,35 @@ script [tf_cnn_benchmarsk.py](https://github.com/tensorflow/benchmarks/blob/mast
...
+users only need to mount under storages to Alluxio once and mount the 
+parent folder of those under storages that contain training data to the local filesystem.
+After the initial mounting, the data becomes immediately available through Alluxio
```
Comment (ZacBlanco) [inline docs/en/compute/Tensorflow.md]: should there be an article before "fuse"? "an", "the", etc..?

File: docs/en/compute/Tensorflow.md [hunk 3]
```diff
@@ -156,35 +156,35 @@ script [tf_cnn_benchmarsk.py](https://github.com/tensorflow/benchmarks/blob/mast
...
 
 By co-locating Tensorflow applications with Alluxio worker, Alluxio caches the remote data
-locally for the future access, providing data locality. Without Alluxio, slow remote
+locally for future access, providing data locality. Without Alluxio, slow remote
 storage may result in bottleneck on I/O and leave GPU resources underutilized. 
 When concurrently writing or reading big files, Alluxio POSIX API can provide significantly better
-performance when running on Alluxio worker node. When the worker node has big enough memory space to 
-host all the training data, Alluxio POSIX API on worker node can provide nearly 2X performance improvement.
+performance when running on Alluxio worker node. Setting up a worker node with memory space to 
+host all the training data can allow Alluxio POSIX API to provide nearly 2X performance improvement.
```
Comment (ZacBlanco) [inline docs/en/compute/Tensorflow.md]: article before "Alluxio POSIX API"?

File: docs/en/compute/Tensorflow.md [hunk 4]
```diff
@@ -156,35 +156,35 @@ script [tf_cnn_benchmarsk.py](https://github.com/tensorflow/benchmarks/blob/mast
...
 Many Tensorflow applications generate a lot of small intermediate files during their
-workflow. Those intermediate files are only useful for a short time and are not needed to be 
-persistent to under storages. If we directly link Tensorflow with remote storages, all the 
-files no matter they are data, intermediate files, or results will write to the remote storage 
-and become persistent. With Alluxio -- a cache layer between the Tensorflow applications and remote storages, 
-users can reduce unneeded remote persistent works and speed up the write/read time.
+workflow. Those intermediate files are only useful for a short time and do not need to be 
+persisted to under storages. If we directly link Tensorflow with remote storages, all files 
+(regardless of the type - data files, intermediate files, results, etc.) will be written to and persisted 
+in the remote storage. With Alluxio -- a cache layer between the Tensorflow applications and remote storages,
```
Comment (ZacBlanco) [inline docs/en/compute/Tensorflow.md]:
```suggestion
in the remote storage. With Alluxio -- a cache layer between the Tensorflow applications and remote storage, 
```

File: docs/en/compute/Tensorflow.md (hunk 2 above)
Comment (tieujason330) [reply docs/en/compute/Tensorflow.md]: updated - added "the"

File: docs/en/compute/Tensorflow.md (hunk 3 above)
Comment (tieujason330) [reply docs/en/compute/Tensorflow.md]: updated - added "the"

File: docs/en/compute/Tensorflow.md (hunk 4 above)
Comment (tieujason330) [reply docs/en/compute/Tensorflow.md]: updated

File: docs/en/compute/Tensorflow.md (hunk 1 above)
Comment (tieujason330) [reply docs/en/compute/Tensorflow.md]: updated

**Comment by ZacBlanco** [comment]
```

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
alluxio-bot, cherry-pick this to branch-2.1 please

**Pull request merged**